### ISSUE TITLE
Config links not showing up on admin/config

### ISSUE THREAD

**Issue by nerdstein** [nerdstein/encrypt#24]
-Should categorize to "security", like other modules (e.g. Password Policy). /admin/config/security/encryption for SettingsForm
-Need to expose the route from within the admin/config page

**Comment by nerdstein** [comment]
Can reference https://github.com/nerdstein/password_policy for example

**Comment by mjl3475** [comment]
Done in Issue-24-add-to-security-section-in-configure, will PR once cleanup is merged into master.

@nerdstein, I had a few questions about it that we can discuss in our meeting.
